### PR TITLE
disable stack tracing for curl operations in e2e

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/9512

### CODE REVIEW

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@stevekuznetsov @liggitt ptal.

summary of the issue this is working around is here:
https://github.com/openshift/origin/issues/9685

**Comment by stevekuznetsov** [comment]
I would definitely NACK this to either:
 - refactor the Bash to be more idiomatic and less fragile https://github.com/openshift/origin/issues/9685#issuecomment-230117742
 - fix the bug in my Bash stracktrace https://github.com/openshift/origin/issues/9685#issuecomment-230118019

Since it doesn't seem like the merge queue is going to make any of these options quicker than any other, and since all of them are fairly similar in size.

**Comment by bparees** [comment]
@stevekuznetsov we can't have the stack trace code causing side effects and expect that everyone who works in the shell scripts will understand the potential side effects and code idiomatically.  the stack trace logic needs to be fixed (and imho, disabled until it is fixed).

**Comment by stevekuznetsov** [comment]
@bparees  fair enough

File: test/end-to-end/core.sh [hunk 1]
```diff
@@ -42,8 +42,14 @@ function wait_for_app() {
   wait_for_url_timed "http://${FRONTEND_IP}:5432" "[INFO] Frontend says: " $((2*TIME_MIN))
 
   echo "[INFO] Testing app"
+  set -x
+  OLD=$OS_USE_STACK_TRACE
+  OS_USE_STACK_TRACE=""
   wait_for_command '[[ "$(curl -s -X POST http://${FRONTEND_IP}:5432/keys/foo -d value=1337)" = "Key created" ]]'
```
Comment (stevekuznetsov) [inline test/end-to-end/core.sh]: In an orthogonal direction, @deads2k do you remember why you argued these should *not* be refactored into `os::cmd::try_until_text` ?

**Comment by bparees** [comment]
@stevekuznetsov i'm inclined to merge this for now to address the flakes(https://github.com/openshift/origin/issues/9512), we can revert the workaround as part of the correct fix for https://github.com/openshift/origin/issues/9685 (and as a bonus, keep the fixed up indenting).

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
(sorry for the self merge but...this 4th of july we declare our independence...of flakes.  :fireworks: )

**Comment by stevekuznetsov** [comment]
Sounds good, I'll get that fixed up ASAP when I'm back in.

**Pull request merged**